examples/src/examples/animation/locomotion.tsx

```diff
@@ -18,7 +18,7 @@ class LocomotionExample {
         </>;
     }
 
-    example(canvas: HTMLCanvasElement, data: any): void {
+    example(canvas: HTMLCanvasElement, deviceType: typeof pc.DEVICETYPE_WEBGPU | typeof pc.DEVICETYPE_WEBGL1 | typeof pc.DEVICETYPE_WEBGL2, data: any): void {
```

Comment:
could we have an enum for this or similar, to allow a single type?

Reply:
maybe use 'any' or 'string' instead? Those pc.DEVICETYPE_*** constants are strings.